server/views/partials/toCourtLink.njk

```diff
@@ -1,14 +1,31 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
 {# Macro to format the CVP link based on known information and the feature switch value #}
 
 {% macro toCourtLink(item, feature) %}
     {% if feature and item.hmctsNumber %}
       <a class="govuk-link govuk-link--no-visited-state govuk-!-text-break-word" href="https://HMCTS{{ item.hmctsNumber }}@meet.video.justice.gov.uk" rel="noreferrer noopener" target="_blank">
         HMCTS{{ item.hmctsNumber }}
       </a>
+      {{ govukButton(
+        {
+          classes: 'govuk-button--inverse govuk-button--copy',
+          html: '<img src="/assets/images/copy-to-clipboard.svg" />',
```

Comment:
This is a hack at the moment pulling in the SVG, needs to go into the component CSS.